Payments/Integration/01_ManageConnection.md

```diff
@@ -0,0 +1,140 @@
+[!!Connections (Settings)](../UserInterface/08a_Connections.md)
+
+# Manage connections to payment service providers (PSP)   
+
+If you want to establish a new connection to a payment service provider (PSP), you can buy, depending on the payment service provider, the needed connection in the app store or license it directly at the payment service provider. By default, Actindo supports the following drivers:
+- Adyen&trade;
+- Amazon pay&trade;
+- EBICS&trade;
+- Stripe&trade;
+- A dummy driver to simulate payments for testing purposes    
+
+The data structure of the connection then provides exactly the data that the PSP needs to exchange data with the Actindo *Payments* module.
+
+In the following, the general settings to create, edit, 
+or disable connections are described. 
+
+
+## Create a PSP connection
+
+Create a connection to a certain payment provider. As soon you have established a connection, an initial synchronization will be triggered, which will transfer the entire data model of the connected payment service provider to Actindo. 
+
+
+#### Prerequisites
+
+At least one connection has been installed.
```

Comment:
```suggestion
At least one connection has been established.
```
Alternativ: created. Ich meine, so haben wir es vorher in anderen Modulen geschrieben.

Reply:
Done